service/policy/db/attribute_fqn.go

```diff
@@ -117,35 +117,58 @@ func (c *PolicyDBClient) AttrFqnReindex(ctx context.Context) (res struct { //nol
 }
 
 func (c *PolicyDBClient) GetAttributesByValueFqns(ctx context.Context, r *attributes.GetAttributeValuesByFqnsRequest) (map[string]*attributes.GetAttributeValuesByFqnsResponse_AttributeAndValue, error) {
-	if r.Fqns == nil || r.GetWithValue() == nil {
+	fqns := r.GetFqns()
+
+	// todo: move to proto validation
```

Comment:
Mind opening a ticket for this? This is a good idea.

Reply:
Yep: https://github.com/opentdf/platform/issues/1651